bcipy/helpers/save.py

```diff
@@ -45,28 +49,13 @@ def init_save_data_structure(data_save_path,
 
         # since this is only called on init, we can make another folder run
         os.makedirs(save_directory)
-        os.makedirs(helper_folder_name)
         os.makedirs(os.path.join(save_directory, 'logs'), exist_ok=True)
 
     try:
-        # Go to folder helpers and list files within it
-        src_files = os.listdir('bcipy/helpers/')
-
-        # Loop through files in helpers and copy important ones over
-        for file_name in src_files:
-
-            # get the full name
-            full_file_name = os.path.join('bcipy/helpers/', file_name)
-
-            # Check that constructed file is a real file and ends in .py
-            if (os.path.isfile(full_file_name)) and '.py' in file_name:
-                # Copy over!
-                copy2(full_file_name, helper_folder_name)
-
```

Comment:
When I was looking into BIDS I meant to ask if these helpers are still needed. I'm glad we're able to clean up the data.